### PR TITLE
Fix ClusterRoleBinding manifests

### PR DESCRIPTION
**Pull request by hiyosi**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

----

This PR  fixes an issue applying ClusterRoleBinding fails.

### CODE REVIEW

File: k8s/rbac.yaml [hunk 1]
```diff
@@ -26,6 +26,7 @@ metadata:
 subjects:
 - kind: ServiceAccount
   name: identityd
+  namespace: default
```
Comment (kevindiu) [inline k8s/rbac.yaml]: It depends on which namespace user deployed, so I think it is better to let it empty or add an comment for it.

Comment (hiyosi) [reply k8s/rbac.yaml]: Yes, I know that. 
This PR, I intended to be able to set up according to the README without changing the manifest.

Comment (mcieplak) [reply k8s/rbac.yaml:29]: Yep, as @kevindiu mentioned, this is dependent on which namespace the user is deployed in so it was intended to be able to change based on what is needed.

Comment (mcieplak) [reply k8s/rbac.yaml:29]: This is something we can configure through the helm template in https://github.com/yahoo/k8s-athenz-identity/pull/14

Comment (mcieplak) [reply k8s/rbac.yaml:29]: I've updated that PR to make this service account field configurable with the namespace the identity service is deployed in.

Comment (hiyosi) [reply k8s/rbac.yaml:29]: Thanks fo reply.
I agree that this will be configurable with helm template.

**Pull request closed without merging**